### DISCUSSION TITLE
Form builder is an open source?

### DISCUSSION THREAD

**Discussion by popuguytheparrot** [react-hook-form/react-hook-form#8791] [Q&A] [answered]
Form builder is an open source?

**Comment by bluebill1049** [comment, marked as answer]
Yes, you can see the source code at the documentation website. it's open source.

**Reply by popuguytheparrot** [reply]
Only on website? No like github project?

**Comment by JoeRoddy** [comment]
The actual source code, for anyone else whose looking:
https://github.com/react-hook-form/documentation/blob/master/src/components/BuilderPage.tsx
